src/helpers.jl

```diff
@@ -5,7 +5,8 @@ function count_rows(data)::Integer
     if Tables.rowaccess(data)
         return length(Tables.rows(data))
     else
-        @error "Can't count the number of rows"
+        # data already Tables.columns
+        return nrows = length(Tables.getcolumn(data, 1))
```

Comment:
Ah, I'm not sure why I didn't do this myself!

Could you remove the `nrows =` for consistency please? 

As I'm unfamiliar with the Tables.jl API it took me a while to understand what it meant, can you maybe clarify the comment to something like:

> If the data only provides column access, return the length of the first column